app/javascript/dashboard/store/modules/accounts.js

```diff
@@ -48,6 +51,20 @@ export const actions = {
       throw new Error(error);
     }
   },
+  create: async ({ commit }, accountInfo) => {
+    commit(types.default.SET_ACCOUNT_UI_FLAG, { isCreating: true });
+    try {
+      const response = await AccountAPI.createAccount(accountInfo);
+      commit(
+        types.default.SET_LINKED_ACCOUNT_ID,
+        response.data.data.linked_account_id
+      );
+      commit(types.default.SET_ACCOUNT_UI_FLAG, { isCreating: false });
```

Comment:
You could return a value here or use a bus emit event instead of having a linked_account_id.